mount/bind_filter_windows.go

```diff
@@ -0,0 +1,327 @@
+package mount
```

Comment:
This makes me wonder if it'd be better suited to be in go-winio. We can lock down the API there, and then hcsshim and here could all consume from it.

Reply:
Yup. Already proposed a ```go-winio``` PR here: https://github.com/microsoft/go-winio/pull/274. Need to write some tests today. Once that is in, I'll update the module and remove these 2 files.

Reply:
Oh sweet! Should've checked 😄